src/jdbc/cubrid/jdbc/jci/UUrlCache.java

```diff
@@ -36,10 +36,29 @@
 public class UUrlCache {
 	private Hashtable<String, UStmtCache> stmt_cache_table;
 	private ArrayList<UStmtCache> stmt_cache_remove_list;
+	private int max_size;
+	private int cache_size;
```

Comment:
cache_size should be an AtomicInteger.

Reply:
another way:
use synchronize at updating?

Reply:
`AtomicInteger` is significantly faster than `synchronized` block.

Reply:
show me a partial code for updating.